.ort.yml

```diff
@@ -75,10 +75,10 @@ excludes:
   scopes:
   - name   : "devDependencies"
     reason : "DEV_DEPENDENCY_OF"
-    comment: "Scope with dependencies only used for development. Not included in released artifacts in the context of this project."
+    comment: "Packages for development only. Not part of released artifacts"
```

Comment:
Commit message: "Align reasons so they are consistent with the output generated by ORT's CLI (GenerateScopeExcludesCommand) and examples."

Reply:
Used @mnonnenmacher suggestion below as improved commit message